pkg/weights/weights.go

```diff
@@ -97,6 +100,10 @@ func isCodeFile(path string) bool {
 	return ext == ".py" || ext == ".ipynb"
 }
 
+func isGitFile(path string) bool {
+	return strings.Contains(path, ".git")
+}
```

Comment:
Any concern about this being too broad? Would it be reasonable to check for `"/.git/"` instead, for example?

Reply:
Resolved in [34000e9](https://github.com/replicate/cog/pull/2087/commits/34000e9a34bdacb799cda1c82fdfffded364772b)